pkg/generate/generate.go

```diff
@@ -128,20 +152,16 @@ func (c *Controller) applyGeneratePolicy(log logr.Logger, policyContext engine.P
 	policy := policyContext.Policy
 	resource := policyContext.NewResource
 	ctx := policyContext.Context
-	// To manage existing resources, we compare the creation time for the default resiruce to be generated and policy creation time
-	processExisting := func() bool {
-		rcreationTime := resource.GetCreationTimestamp()
-		pcreationTime := policy.GetCreationTimestamp()
-		return rcreationTime.Before(&pcreationTime)
-	}()
+	// To manage existing resources, we compare the creation time for the default resource to be generated and policy creation time
 
+	processExisting := false
```

Comment:
Why set the default to false? If the resource exists before the policy is created, we do not want to generate anything.

Reply:
but in new case if we have a namespace and we created a policy after policy creation we add label that means resource exist but we want to create a new resource. I can remove the processExisting variable if you want